pkg/etcd/client.go

```diff
@@ -142,3 +151,37 @@ func (c *Client) Health(ctx context.Context) error {
 	return err
 
 }
+
+// Write tries to write value with a given key and returns indicator if write operation succeed
+func (c *Client) Write(ctx context.Context, key string, value string, ttl int64) (bool, error) {
```

Comment:
I could add it, but it just adds additional conversion layer between time.Duration and int64, as the etcd client accepts int64.

Reply:
Sorry, haven't answered the main question. The ttl is a duration of record lifetime.

Reply:
A duration would have seemed more idiomatic. Not sure if this is seconds, milliseconds, nanoseconds. The Write function would have had to do the conversion, but it would have provided more clarity and ease of use at the call site.

```go
c.Write(ctx, "foo", "bar", 1*time.Minute)
// versus
c.Write(ctx, "foo", "bar", 60 /* seconds */)
```